documentation/advanced/tutorial-all-vaadin-properties.asciidoc

```diff
@@ -95,4 +95,10 @@ Please consult <<tutorial-i18n-localization#, I18N localization>> documentation.
 |disable.automatic.servlet.registration |
 Configuration name for the parameter that determines if Flow should automatically register servlets needed for the application to work.
 
+|vaadin.compatibilityMode |
+It's a boolean property that forces the Vaadin 14 compatibility mode (Bower
```

Comment:
Use one line per sentence.

I'd also use style: "If set to `true`,  ...." Now people might try to set it to `on` or `off`. Also, mention what is the default as it is not very intuitive in this particular case.